spec/onc_certification_g10_test_kit/bulk_data_group_export_spec.rb

```diff
@@ -247,7 +247,9 @@ def run(runnable, inputs = {})
       expect(result.result_message).to eq('Complete Status response did not contain "transactionTime" as required')
     end
 
-    it 'passes when server returns "202 Accepted" and response body contains required attributes' do
+    it 'passes when server returns "202 
+    
+    ed" and response body contains required attributes' do
```

Comment:
```suggestion
    it 'passes when server returns "202 Accepted" and response body contains required attributes' do
```

Reply:
Actually, why is this changed at all?  Just a rogue change?

Reply:
Idk.  Pushed fix.